src/core/tests/auth.json

```diff
@@ -0,0 +1,26 @@
+{
+  "cookies": [],
+  "origins": [
+    {
+      "origin": "http://localhost:8081",
+      "localStorage": [
+        {
+          "name": "main",
+          "value": "{\"user\":{\"id\":\"\",\"name\":\"\",\"organization_name\":\"\",\"permissions\":[]},\"vertical_view\":false,\"drawerVisible\":true,\"itemCountTotal\":0,\"itemCountFiltered\":0,\"coreAPIURL\":\"http://127.0.0.1:5000/api\",\"gitInfo\":\"\",\"buildDate\":\"2023-11-18T16:32:09.587Z\",\"sentryDSN\":\"\",\"notification\":{\"message\":\"\",\"type\":\"\",\"show\":false}}"
+        },
+        {
+          "name": "authenticator",
+          "value": "{\"jwt\":\"\",\"login_uri\":\"/login\",\"logout_uri\":\"/logout\",\"external_login_uri\":false,\"external_logout_uri\":false,\"user\":{},\"sub\":\"\",\"exp\":0}"
+        },
+        {
+          "name": "ACCESS_TOKEN",
```

Comment:
We should try to replace this and try to inject the token from the pytest fixture.

Reply:
We could definitely consider this. However, this is ONLY needed when generating the code for tests using the Codegen tool.